tsconfig.package.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Do we need to have both this and tsconfig.json next to each other? What's the behavior of just folding tsconfig.json and this together? Do we even need a tsconfig.json in the repo root or could we stash the base config away in `/common`?

Reply:
The problem is that we have CI checks that do npm install on the root (I think for the management libs) and they use gulp... which breaks if we don't use commonjs as the target because ts-gulp doesn't support this right. It always needs a tsconfig.json in the root.

Maybe this is something we can take up with eng sys at some point.